battleShip.py

```diff
@@ -0,0 +1,280 @@
+import random
+import pickle
+
+
+class Ship:
+    def __init__(self, size): # Constructor for the Ship class
+        self.size = size
+
+class Player:
+    def __init__(self, name): # Constructor for the Player class
+        self.name = name
+        self.ships = []
+        self.destroyedShips = set()
+        self.grid = []
+
+    def createGrid(self, size): # Create a grid for the player
```

Comment:
please use pep8/python recommended method/variable name style i.e ```createGrid``` should be ```create_grid```

Reply:
ok